WordPress/Classes/Services/MediaService.m

```diff
@@ -16,140 +16,8 @@
 
 NSErrorDomain const MediaServiceErrorDomain = @"MediaServiceErrorDomain";
 
-@interface MediaService ()
-
-@property (nonatomic, strong) MediaThumbnailService *thumbnailService;
-
-@end
-
 @implementation MediaService
 
-- (instancetype)initWithManagedObjectContext:(NSManagedObjectContext *)context
-{
-    self = [super initWithManagedObjectContext:context];
-    if (self) {
-        _concurrentThumbnailGeneration = NO;
-    }
-    return self;
-}
-
-#pragma mark - Creating media
-
-- (Media *)createMediaWith:(id<ExportableAsset>)exportable
```

Comment:
⚠️ Potential issue: this functions returns a `Media` instance with no guarantee what context object this instance is bound to.